src/js/components/Notification/__tests__/Notification-test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import 'jest-styled-components';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
+
+import { axe } from 'jest-axe';
+import { render, cleanup } from '@testing-library/react';
+
+import { Grommet, Notification } from '../..';
+
+describe('Notification', () => {
+  afterEach(cleanup);
+
+  test('should have no accessibility violations', async () => {
+    const { container } = render(
+      <Grommet>
+        <Notification />
+      </Grommet>,
+    );
+
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
Can we add a `toast` test too?

Reply:
i added a test, but not sure if it's a good one. might need some guidance with writing better tests because i haven't written one for a component in a while